packages/react/src/ReactElementValidator.js

```diff
@@ -365,13 +369,20 @@ export function jsxWithValidation(
     }
   }
 
-  if (hasOwnProperty.call(props, 'key')) {
```

Comment:
moved this inside the __DEV__ block. It wasn't being DCEed, so that gets us a few bytes back too.

Reply:
Wait, wasn’t this whole file supposed to be dev only? Are you saying this function is used in prod?

Reply:
Oh hmm you're right. I only glanced at sizebot, probably not relevant.